components/progresscircle/stories/progresscircle.stories.js

```diff
@@ -1,6 +1,6 @@
 import { Sizes } from "@spectrum-css/preview/decorators";
 import { disableDefaultModes } from "@spectrum-css/preview/modes";
-import { isIndeterminate, size, staticColor } from "@spectrum-css/preview/types";
+import { isIndeterminate, size } from "@spectrum-css/preview/types";
```

Comment:
Let's get the `staticColor` import back! Yay!

`import { isIndeterminate, size, staticColor } from "@spectrum-css/preview/types";`

That means we can basically revert these `staticColor` control changes as well. In the controls, we should just be able to do `staticColor`, since we need both the black and white options now too. 👍 

<img width="132" alt="Screenshot 2025-01-08 at 4 14 42 PM" src="https://github.com/user-attachments/assets/84c55955-931e-46c5-a9e5-b4166869affb" />